vmdk_plugin/plugin.go

```diff
@@ -208,12 +209,55 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // (until Mount is called).
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
-	err := d.ops.Create(r.Name, r.Options)
+	// Use default fstype if not specified
+	if _, ok := r.Options["fstype"]; ok == false {
```

Comment:
nit: `ok == false ` looks really strange, I think the name should be 'result'. `result == false` is easy to digest :-)